### DISCUSSION TITLE
[Question] should I git init vault or one level higher?

### DISCUSSION THREAD

**Discussion by simkimsia** [dendronhq/dendron#1831] [General]
Typically I git init my main Dendron vault.

Inside that I also added other vaults from project workspaces.

Love this setup.

Recently moving from 2014mbp to 2021mbp so inevitably I use git clone

Then I realised my links to other vaults are dead in the new mbp 

Because

A) I haven't git clone the project repos yet
B) I need to re-add the vaults back
C) and B happens because dendron.yml itself is not part of the repository

My question is should I as a practice have only the vault in git repo or vault and adjacent files?

Thank you

**Comment by airtonix** [comment]
> Recently moving from 2014mbp to 2021mbp so inevitably I use git clone

why is that inevitable?
